x-pack/plugins/observability_solution/inventory/public/services/telemetry/telemetry_client.ts

```diff
@@ -34,12 +33,6 @@ export class TelemetryClient implements ITelemetryClient {
     this.analytics.reportEvent(TelemetryEventTypes.ENTITY_INVENTORY_SEARCH_QUERY_SUBMITTED, params);
   };
 
-  public reportEntityInventoryEntityTypeFiltered = (
-    params: EntityInventoryEntityTypeFilteredParams
-  ) => {
-    this.analytics.reportEvent(TelemetryEventTypes.ENTITY_INVENTORY_ENTITY_TYPE_FILTERED, params);
-  };
-
```

Comment:
We don't need this telemetry info anymore?

Reply:
It's no longer as straightforward as it was before to get the selected entity types now using control groups. The selected items are in the URL anyway.

Reply:
Got it. should we align this with Roshan?

Reply:
cc: @roshan-elastic

Reply:
Hey @cauemarcondes @crespocarlos - will we still be able to populate the telemetry like this?

```json
{
	"timestamp": "2024-11-12T11:20:08.790Z",
	"event_type": "Entity Inventory Search Query Submitted",
	"context": {
	...
	},
	"properties": {
		"kuery_fields": ["entity.name"],
		"entity_types": ["service"],
		"action": "refresh"
	}
}
```

Not super fussed if we explicitly track the control filter itself but if someone filters by `entity.type` either via the control filter via a query - we need to track this.

Reply:
No, we won't be able to. The control group has a complex data structure and it's not possible to easily extract the selected options. It also can select item but exclude them from the query.